core/eolearn/core/eodata_io.py

```diff
@@ -106,8 +106,10 @@ def save_eopatch(
     # Data must be collected before any tinkering with files due to lazy-loading
     data_for_saving = list(_yield_features_to_save(eopatch, eopatch_features, patch_location))
 
-    if overwrite_permission is OverwritePermission.OVERWRITE_PATCH and patch_exists:
-        _remove_old_eopatch(filesystem, patch_location)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=EODeprecationWarning)
```

Comment:
is this ignored because the warning will be shown already by using the permission in the first place? was the warning triggering twice?

Reply:
this is ignored because here WE are using the constant. Even if the user provide `ADD_ONLY` we check it against `OVERWRITE_PATCH` which triggers the warning.